### PR TITLE
Fix bug in mmCIF parser

### PR DESCRIPTION
**Pull request by sbliven**
Loading cif files with spacegroup info failed due to typo

### CODE REVIEW

File: js/ngl/parser.js [hunk 1]
```diff
@@ -2782,8 +2782,8 @@ NGL.CifParser.prototype = NGL.createObject(
 
             var symmetry = cif.symmetry;
 
-            var sGroup = symmetry[ "space_group_name_H-M" ];
-            if( sGroup[0] === sGroup[ sGroup.length-1 ] &&
+            var sGroup = symmetry[ "space_group_name_H_M" ];
```
Comment (josemduarte) [inline js/ngl/parser.js]: The field in mmCIF has a hyphen between H and M i.e. `space_group_H-M`. Does it work with underscore?

**Comment by arose** [comment]
Hmm, the mmcif dict says it is a [hyphen]( http://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v40.dic/Items/_symmetry.space_group_name_H-M.html). I am ok with adding the underscore variant if there are files out there needing it.

**Comment by hainm** [comment]
> I am ok with adding the underscore variant if there are files out there needing it.

It's better to force people to follow standard.

**Comment by sbliven** [comment]
The cif file i'm using was generated by biojava, so if there are errors we should fix them in biojava not ngl

**Pull request closed without merging**